Library Catalog (Encore).js

```diff
@@ -36,204 +35,183 @@
-	Zotero.Utilities.processDocuments(urls, function(newDoc) {
-		scrape(marc, newDoc);
-	}, function() { Zotero.done() });
+				newItem.repository = domain[1].replace(/encore\./, "");
+				// there is too much stuff in the note field - or file this as an abstract?
+				newItem.notes = [];
+				newItem.complete();
+			});
+		});
+	}
 }
 
 /** BEGIN TEST CASES **/
-var testCases = [
-	{
+var testCases = [{
 		"type": "web",
-		"url": "http://encore.colorado.edu/iii/encore/search?formids=target&lang=eng&suite=def&reservedids=lang%2Csuite&submitmode=&submitname=&target=thelen&Search.x=0&Search.y=0",
+		"url": "http://sallypro.sandiego.edu/iii/encore/search/C__Rb3558162__Stesting__Orightresult__U__X6?lang=eng&suite=cobalt#resultRecord-b3558162",
 		"items": "multiple"
 	},
 	{
 		"type": "web",
-		"url": "http://encore.coalliance.org/iii/encore/search/C|Sthelen|Orightresult|U1?lang=eng",
-		"items": "multiple"
-	},
-	{
-		"type": "web",
-		"url": "http://encore.colorado.edu/iii/encore/record/C__Rb5910060__Sthelen__P0%2C3__Orightresult__X4?lang=eng&suite=cobalt",
-		"items": [
-			{
-				"itemType": "book",
-				"creators": [
-					{
-						"firstName": "Marcel",
-						"lastName": "Thelen",
-						"creatorType": "editor"
-					},
-					{
-						"firstName": "F.",
-						"lastName": "Steurs",
-						"creatorType": "editor"
-					}
-				],
-				"notes": [],
-				"tags": [
-					"Language and languages",
-					"Terms and phrases"
-				],
-				"seeAlso": [],
-				"attachments": [],
-				"ISBN": "9789027223371",
-				"title": "Terminology in everyday life",
-				"place": "Amsterdam, The Netherlands : Philadelphia, Pa",
-				"publisher": "John Benjamins Pub. Co",
-				"date": "2010",
-				"numPages": "271",
-				"series": "Terminology and lexicography research and practice",
-				"seriesNumber": "v. 13",
-				"callNumber": "P305 .T4437 2010",
-				"libraryCatalog": "colorado.edu Library Catalog"
-			}
-		]
+		"url": "http://sallypro.sandiego.edu/iii/encore/record/C__Rb1516899__Stesting__P0%2C2__Orightresult__U__X6?lang=eng&suite=cobalt",
+		"items": [{
+			"itemType": "thesis",
+			"title": "Testing a theoretical model of critical thinking and cognitive development",
+			"creators": [{
+					"firstName": "Jane",
+					"lastName": "Rapps",
+					"creatorType": "author"
+				},
+				{
+					"lastName": "University of San Diego",
+					"creatorType": "contributor",
+					"fieldMode": true
+				}
+			],
+			"date": "1998",
+			"callNumber": "BF441 .R3 1998",
+			"libraryCatalog": "sallypro.sandiego.edu Library Catalog",
+			"numPages": "199",
+			"attachments": [],
+			"tags": [{
+					"tag": "Constructivism (Education)"
+				},
+				{
+					"tag": "Critical thinking"
+				},
+				{
+					"tag": "Decision making"
+				},
+				{
+					"tag": "Dissertations"
+				},
+				{
+					"tag": "Education"
+				},
+				{
+					"tag": "Nursing"
+				},
+				{
+					"tag": "Nursing"
+				},
+				{
+					"tag": "Nursing"
+				},
+				{
+					"tag": "Nursing"
```

Comment:
Look at the MARC here: http://sallypro.sandiego.edu/iii/encore/record/C__Rb1516899__Stesting__P0,2__Orightresult__U__X6?lang=eng&suite=cobalt&marcData=Y
in the 650 fields:
650  0 Nursing|xEducation 
650  0 Critical thinking 
650  0 Constructivism (Education) 
650  0 Nursing|xDecision making 
650  0 Nursing|xPhilosophy 

MARC imports both parts of the field as tag -- maybe that's not right? Or we should save the category in the 0 field and check against it to avoid duplication? In any case, this is a MARC issue, I'd like to address it separately

Reply:
Okay, I see.

Although, I thought that duplicate tags were technically not possible. At least manually I cannot enter twice the same tag.

Reply:
Turns out they only show up in the tests, not in Zotero itself.

Reply:
ok